### PR TITLE
test yaml crisprme

### PR DESCRIPTION
**Pull request by samuelecancellieri**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by samuelecancellieri** [comment]
@BiocondaBot please add label

**Comment by samuelecancellieri** [comment]
I'm obtaining an error in the linting phase, but I can't understand what is wrong.
Thanks

File: recipes/crisprme/meta.yaml [hunk 1]
```diff
@@ -10,12 +10,35 @@ source:
...
+    - flask=1.1.2=pyh9f0ad1d_0
+    - flask-caching=1.7.1=py_0
+    - flask-compress=1.5.0=pyh9f0ad1d_0
+    - fontconfig=2.13.1=h86ecdb6_1001
+    - freetype=2.10.1=he06d7ca_0
+    - future=0.18.2=py38h32f6830_1
+    - gettext=0.19.8.1=hc5be6a0_1002
+    - gunicorn=20.0.4=py38_0
+    - pip
+    - pip:
```
Comment (dpryan79) [inline recipes/crisprme/meta.yaml:34]: We don't support pip-installed dependencies.

Comment (dpryan79) [reply recipes/crisprme/meta.yaml:34]: Also, is there a reason you're specifying exact packages as dependencies? Even exact versions is a bit restrictive.

Comment (samuelecancellieri) [reply recipes/crisprme/meta.yaml:34]: Hi,
the version in the packages is to be sure any problem is avoided, dash can be very problematic to run without the exact package.
Regarding pip, we will try to check if these dependencies can be satisfied differently.

Thanks

**Comment by samuelecancellieri** [comment]
@BiocondaBot please add label

File: recipes/crisprme/meta.yaml (hunk 1 above)
Comment (samuelecancellieri) [reply recipes/crisprme/meta.yaml]: Hi, can you review the new file? I've tested it with less restrictive version and no pip packages
thanks

**Review by lucapinello — APPROVED** [review]

**Pull request merged**